CityPuzzle/Classes/IUserVerifier.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CityPuzzle.Classes
+{
+    public interface IUserVerifier
+    {
+        bool CPass(string name, string pass);
```

Comment:
Add comments to specify what these methods do as theyr names ar not very descriptive